imported/spirv/spvgen.h

```diff
@@ -412,7 +409,8 @@ static const char* SpvGeneratorName = "spvgen.so";
 // =====================================================================================================================
 // Initialize SPIR-V generator entry-points
 // This can be called multiple times in the same application.
-bool InitSpvGen()
+bool InitSpvGen(
+    const char* pSpvGenDir)   // Directory to load SPVGEN library from, or "" to use OS's default search path
```

Comment:
// -> // [in]

Reply:
Now fixed in https://github.com/GPUOpen-Drivers/spvgen/pull/3
I will update this commit a bit later on.

Reply:
Fixed here as well now.